R/diagnostics.R

```diff
@@ -117,24 +126,22 @@ pareto_k_table <- function(x) {
 print.pareto_k_table <- function(x, digits = 1, ...) {
   count <- x[, "Count"]
 
-  if (sum(count[2:4]) == 0) {
-    cat("\nAll Pareto k estimates are good (k < 0.5).\n")
+  if (sum(count[2:3]) == 0) {
+    cat(paste0("\nAll Pareto k estimates are good (k < ", threshold, ").\n"))
```

Comment:
One of the reasons for the test failures is that `threshold` doesn't exist here inside the print method for the table. We can grab it from the new Threshold column of `x` (the table). That column will always be a constant (have the same value in all rows), right?

Reply:
Also we should round threshold here, otherwise it will print something like `All Pareto k estimates are good (k < 0.666666666666667)`. How many digits do you want to print?

Reply:
Fixed